be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/SocialMediaHandler.scala

```diff
@@ -40,6 +40,13 @@ class SocialMediaHandler(dbRef: => AskableActorRef) extends MessageHandler {
 
   private def generateSocialChannel(lao_id: Hash): Channel = Channel(Channel.ROOT_CHANNEL_PREFIX + lao_id + Channel.SOCIAL_MEDIA_CHIRPS_PREFIX)
 
+  // the following function encodes the data and broadcast it
+  private def broadcastData[T](rpcMessage: JsonRpcRequest, data: T, channel: Channel, broadcastChannel: Channel): GraphMessage = {
+    val encodedData: Base64Data = Base64Data.encode(data.toString)
```

Comment:
in doing this refactoring, we went from `data.toJson.toString` to `data.toString`, was that forgotten or is it intentional ?